packages/xrpl/test/createMockRippled.ts

```diff
@@ -109,6 +128,8 @@ export default function createMockRippled(port: number): MockedWebSocketServer {
           mock.testCommand(conn, request)
         } else if (request.command in mock.responses) {
           conn.send(createResponse(request, mock.getResponse(request)))
+        } else if (request.command === 'server_info') {
+          server_info(conn, request)
```

Comment:
The mocks need to be set in the test itself, not here - this is going to add the mock to every test that uses `server_info`.

Reply:
Agree. I can just add a `server_info` response to setupClient on connected instead of doing this.